.github/workflows/e2e-test.yml

```diff
@@ -128,7 +141,9 @@ jobs:
       E2E_PHP_VERSION: ${{ matrix.php }}
       E2E_GROUP:  ${{ matrix.test_groups }}
       E2E_BRANCH: ${{ matrix.test_branches }}
-      SKIP_WC_BLOCKS_TESTS: 1 # skip running blocks tests
+      SKIP_WC_SUBSCRIPTIONS_TESTS: ${{ matrix.test_groups == 'blocks' && '1' || '' }}
+      SKIP_WC_ACTION_SCHEDULER_TESTS: ${{ matrix.test_groups == 'blocks' && '1' || '' }}
+      SKIP_WC_BLOCKS_TESTS: ${{ matrix.skip_blocks }}
```

Comment:
I wonder why we need `skip_blocks` while we can use the following validation, similar to `SKIP_WC_SUBSCRIPTIONS_TESTS` and `SKIP_WC_ACTION_SCHEDULER_TESTS` above: 


`${{ matrix.test_groups == 'blocks' && '1' || '' }}`

By removing `skip_blocks`, we do not need to make any change to matrix entries for this section `Add WC 7.7.0 with PHP 7.3 only (legacy - NO blocks tests)`

Reply:
Yes, great catch. I removed the skip_blocks in [530a068](https://github.com/Automattic/woocommerce-payments/pull/11005/commits/530a068f49d4fcb036e69cf19152a3b91583ad64) thanks!